releasenotes/notes/remote-config-enabled-by-default-9f6d968f7995915d.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
```

Comment:
I'm not sure if `enhancement` is the proper category for this change given the impact but I'll leave it up to you on what level of visibility to elevate this message to.

Reply:
You're right, bumped the release note to `features` instead